### PR TITLE
Add pprof endpoint for easier debugging

### PR DESCRIPTION
**Pull request by deejay1**
Also make it usable for forking and travis and update
golang test versions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,12 @@ sudo: false
 
 language: go
 
+go_import_path: github.com/prometheus/graphite_exporter
+
 go:
-- 1.5.4
-- 1.6.2
+ - 1.7.x
```
Comment (grobie) [inline .travis.yml]: I think 1.9.x would be enough, we don't build graphite-exporter with older golang anyway.

Comment (deejay1) [reply .travis.yml]: Hmm, maybe I should update the circle.yml file also? I just noticed it has 1.6-base as a test image

Comment (brian-brazil) [inline .travis.yml]: This should be consistent with Circle, and we only really need whatever we're currently building with.

Comment (SuperQ) [reply .travis.yml]: Yup, please do update it.

Comment (deejay1) [reply .travis.yml]: done

Comment (deejay1) [reply .travis.yml]: done

**Comment by SuperQ** [comment]
It would be nice to have the `.travis.yml` also build with `- master` like we do in `node_exporter`.

File: .travis.yml [hunk 2]
```diff
@@ -2,9 +2,10 @@ sudo: false
 
 language: go
 
+go_import_path: github.com/prometheus/graphite_exporter
```
Comment (SuperQ) [inline .travis.yml]: This should be below `go:` like we have it in other `.travis.yml` files.

Comment (deejay1) [reply .travis.yml]: Done I hope ;)

**Comment by SuperQ** [comment]
Great! Thanks!

**Pull request merged**